app/instance-initializers/axe-component.js

```diff
@@ -8,27 +10,32 @@ let hasRan = false;
 export function initialize(application) {
   if (hasRan) { return; }
 
+  const appConfig = ENV['ember-a11y-testing'] || {};
+  const { componentOptions: { axeOptions, turnAuditOff, axeCallback } = {} } = appConfig;
+
   Ember.Component.reopen({
     /**
      * An optional callback to process the results from the a11yCheck.
      * @public
      * @type {Function}
      */
-    axeCallback: undefined,
+    axeCallback,
 
     /**
      * An optional options object to be used in a11yCheck.
      * @public
      * @type {Object}
      */
-    axeOptions: undefined,
+    axeOptions,
 
     /**
      * Turns off the accessibility audit during rendering.
+     * Defaults to false if not set in the application's configuration.
+     *
      * @public
      * @type {Boolean}
      */
-    turnAuditOff: false,
+    turnAuditOff: typeof turnAuditOff === 'boolean' ? turnAuditOff : false,
```

Comment:
That’s a REALLY interesting (and good) point. 

My approach here was considering a similar reaction on the opposite end of the spectrum: someone introducing `ember-a11y-testing` into existing application, and all of a sudden having a hundred components being audited in ways they haven’t yet had a chance to control.

So I'm split -- but largely on your side now 😃. I can totally get behind the notion of not having a global switch, and perhaps exploring more config options if we get more user feedback around the matter in the future.